packages/examples/function-http-raw-response/index.ts

```diff
@@ -14,4 +15,4 @@ class Example {
     }
 }
 
-export default Example.rawResponse;
+export default toAzureFunction(() => new Example());
```

Comment:
I agree that it would be cool, but I could not find a solution for that so far.

When exporting the class instance, I get this azure functions error: 
`Exception: Worker was unable to load function function-http-raw-response: 'The resolved entry point is not a function and cannot be invoked by the functions runtime. Make sure the function has been correctly exported.'`

The runtime expects that a callable function is exported.

Any idea or suggestion?